src/lib.rs

```diff
@@ -74,13 +93,14 @@ pub mod sonic_pc;
 /// The construction is detailed in [[BCMS20]][pcdas].
 ///
 /// [pcdas]: https://eprint.iacr.org/2020/499
+// TODO: add "Prepared" to marlin_pc
 pub mod ipa_pc;
 
 /// `QuerySet` is the set of queries that are to be made to a set of labeled polynomials/equations
 /// `p` that have previously been committed to. Each element of a `QuerySet` is a `(label, query)`
 /// pair, where `label` is the label of a polynomial in `p`, and `query` is the field element
 /// that `p[label]` is to be queried at.
-pub type QuerySet<'a, F> = BTreeSet<(String, F)>;
+pub type QuerySet<'a, F> = BTreeSet<(String, (String, F))>;
```

Comment:
Can you please also update the documentation? What's the extra `String` term for now? Should this be a `BTreeMap` instead?

Reply:
I will also update the documentation later today to reflect this change.

The extra String is the "meaningful label" for the point (F), like "beta", which is necessary for the constraint world to be data-independent.

Reply:
Updated the documentation. Will be pushed in a PR soon.